src/main/kotlin/net/leanix/githubagent/dto/ManifestFilesDTO.kt

```diff
@@ -0,0 +1,12 @@
+package net.leanix.githubagent.dto
+
+data class ManifestFilesDTO(
+    val repositoryId: String,
+    val repositoryFullName: String,
+    val manifestFiles: List<ManifestFileDTO>
+)
+
+data class ManifestFileDTO(
+    val path: String,
+    val content: String?,
```

Comment:
Should the content be mantadory? an empty file would be an empty String. Not sure about this

Reply:
good point, I'll fix it